tools/coverage/main.go

```diff
@@ -116,9 +116,10 @@ func main() {
 				"fail presubmit workflow intentionally", *covThresholdFlag)
 		}
 	case "periodic":
-		log.Printf("job type is %v, producing testsuite xml...\n", jobType)
-		testgrid.ProfileToTestsuiteXML(localArtifacts, *covThresholdFlag)
+		log.Printf("job type is %v\n", jobType)
```

Comment:
It's not only for debugging. It was suggested by Kubernetes team to produce testgrid xml for pull request and post-submit jobs as well. 
It's not described in PR title, because of space limit.  It's described in PR's description, however.

Reply:
You should split this PR into 2 then.